micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/QuickStart/Config.js

```
@@ -54,27 +55,201 @@ const QuickSetupConfigComponent = ({ onSelect, formData, control, formState, ...
+          answer: "SANDBOX_FAQ_ANS_THREE",
+        },
+        {
+          question: "SANDBOX_FAQ_QUES_FOUR",
+          answer: "SANDBOX_FAQ_ANS_FOUR",
+        },
+      ],
+    },
+  ];
+
+  // return <QuickSetupComponent config={QuickSetupConfig}></QuickSetupComponent>;
+  return <QuickSetup cardConfig={cardConfig}></QuickSetup>;
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)